Intersect.Server/Networking/PacketHandler.cs

```diff
@@ -1643,7 +1643,14 @@ public void HandlePacket(Client client, Player player, [NotNull] OfferTradeItemP
         //RevokeTradeItemPacket
         public void HandlePacket(Client client, Player player, [NotNull] RevokeTradeItemPacket packet)
         {
-            player?.RevokeItem(packet.Slot, packet.Quantity);
+            if (player.Trading.Counterparty.Trading.Accepted)
```

Comment:
There's a null handler on the original method, and I see you copied that over. But I don't see it in your additions?

Reply:
I didn't think there would be a NRE here, assuming the trade is still happening and Accepted is a bool.

Should i add ```
            if (player == null || player.Trading.Counterparty == null)
            {
                return;
            }``` in front of it?

Reply:
I'm not sure, in theory you'd assume you never could hit an NRE there.. unless a player somehow manages to not be on the server anymore. WHich is rare, but not impossible.

It was mostly a discrepency I caught reading your changes, mind. But personally, I'd try and keep it consistent. If the old code checks for it, the new should do the same.